app/models/log.rb

```diff
@@ -50,6 +50,7 @@ def process_uprn_change!
 
       presenter = UprnDataPresenter.new(service.result)
 
+      self.uprn_known = 1
```

Comment:
How come this needs to be set separately? I expected it would have to be filled in if the uprn is given